Adyen/Integration/02_ManageAdyenConnection.md

```diff
@@ -0,0 +1,161 @@
+# Manage Adyen connections
+
+In the following, the procedures to create, edit, configure, enable, or disable Adyen&trade; connections in the *Payments* module are described.  
+
+Note the following:   
+- If you have more than one merchant account in Adyen&trade; that you want to manage with Actindo, you must create a connection for each.   
+- Adyen&trade; offers the possibility to work with two separate environments, a staging environment for testing purposes and a live environment for production. You must configure the connection for each environment separately.
+- It is recommended that you manage the Adyen&trade; settings in parallel, so that you can enter the connection data alternately. Note that you can no longer copy the credentials after you have saved the Adyen&trade; or Payments settings. For detailed information, see [Manage Adyen settings](01_ManageAdyenSettings.md).
+
+
+## Create Adyen connection
+Create a connection to the Adyen&trade; payment service provider.
+
+#### Prerequisites
+
+- You have an Adyen&trade; customer account to log in to Adyen&trade;.
+- You are logged in to Adyen&trade;.
```

Comment:
Muss man zwischen Adyen customer account und Adyen backend hier auch unterscheiden? In Manage Adyend settings reden wir in der Regel über Adyen backend. Sind die Login-Details gleich oder nicht unbedingt?